### PR TITLE
Small perfomance and memory usage improvements

### PR DESCRIPTION
**Pull request by DamirAinullin**
Small perfomance and memory usage improvements

### CODE REVIEW

File: TaskEditor/EventViewerControl.cs [hunk 1]
```diff
@@ -39,9 +40,9 @@ public TaskEvent TaskEvent
 					computerText.Text = taskEvent.EventRecord.MachineName;
 					detailsText.Text = taskEvent.EventRecord.FormatDescription();
 					eventIdText.Text = taskEvent.EventRecord.Id.ToString();
-					System.Collections.Generic.List<string> keywords = new System.Collections.Generic.List<string>();
-					foreach (string s in taskEvent.EventRecord.KeywordsDisplayNames)
-						keywords.Add(s);
+					System.Collections.Generic.List<string> keywords =
```
Comment (dahall) [inline TaskEditor/EventViewerControl.cs]: Will you change lines 43-46 to the following?
```suggestion
keywordsText.Text = string.Join(", ", taskEvent.EventRecord.KeywordsDisplayNames.ToArray());
```

**Review by dahall — CHANGES_REQUESTED** [review]
I have just the single comment for EventViewerControl. Once you make that change and submit, I'll accept the PR. Thanks.

File: TaskEditor/EventViewerControl.cs (hunk 1 above)
Comment (DamirAinullin) [reply TaskEditor/EventViewerControl.cs]: I just added initial capacity for the List.

Comment (dahall) [reply TaskEditor/EventViewerControl.cs]: I saw that. My suggestion was based on not even needing to create the list in the first place and directly supplying `string.Join` with a string array created via `Linq`.

**Pull request merged**